From e891311096b0b44a45e3e74baaa4f614c16d4bed Mon Sep 17 00:00:00 2001
From: Andre Amegah <andrex.amegah@intel.com>
Date: Mon, 29 Apr 2013 17:35:56 +0300
Subject: [PORT FROM MAIN] Telephony: Handle LAUNCH_BROWSER_ERROR

BZ: 60448

Upon, LAUNCH BROWSER - if not already launched
proactive command, if the browser is already
launched, then the response will be received
with resCode set to LAUNCH_BROWSER_ERROR.
This case was not handled.

Change-Id: I3a3adfc7d0bee9ab9b5c5a10e69f2867368e0776
Orig-Change-Id: I0a7319ea308448ea4f5e4cd5481c895ba2cf322a
Category: aosp improvement
Domain: Telephony-Telephony-FWK
Origin: internal
Upstream-Candidate: yes
Orig-Change-Id: I3dcb7eaea962a35def7a9691e280737361fae5c2
Signed-off-by: Andre Amegah <andrex.amegah@intel.com>
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
---
 .../android/internal/telephony/cat/CatService.java |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/java/com/android/internal/telephony/cat/CatService.java b/src/java/com/android/internal/telephony/cat/CatService.java
index bd7c554..be39881 100644
--- a/src/java/com/android/internal/telephony/cat/CatService.java
+++ b/src/java/com/android/internal/telephony/cat/CatService.java
@@ -840,6 +840,7 @@ public class CatService extends Handler implements AppInterface {
         case PRFRMD_NAA_NOT_ACTIVE:
         case PRFRMD_TONE_NOT_PLAYED:
         case TERMINAL_CRNTLY_UNABLE_TO_PROCESS:
+        case LAUNCH_BROWSER_ERROR:
             switch (type) {
             case SET_UP_MENU:
                 helpRequired = resMsg.mResCode == ResultCode.HELP_INFO_REQUIRED;
-- 
1.7.4.1

